bin/cml.js

```diff
@@ -29,59 +61,52 @@ const configureLogger = (level) => {
   });
 };
 
+const setupTelemetry = async (opts) => {
+  const { cml, cmlCommand: action } = opts;
+  opts.telemetryEvent = await jitsuEventPayload({ action, cml });
+};
+
 const runPlugin = async ({ $0: executable, command }) => {
-  try {
-    if (command === undefined) throw new Error('no command');
-    const path = which.sync(`${basename(executable)}-${command}`);
-    const parameters = process.argv.slice(process.argv.indexOf(command) + 1); // HACK
-    process.exit(await pseudoexec(path, parameters));
-  } catch (error) {
-    yargs.showHelp();
-    winston.debug(error);
-  }
+  if (command === undefined) throw new Error('no command');
+  const path = which.sync(`${basename(executable)}-${command}`);
+  const parameters = process.argv.slice(process.argv.indexOf(command) + 1); // HACK
+  await pseudoexec(path, parameters);
 };
 
-const handleError = (message, error) => {
+const handleError = async (message, error, yargs) => {
```

Comment:
what is the difference between message and error/error.message ?

Reply:
Sorry I do not follow you 🙏